src/screens/UserPortal/LeaveOrganization/LeaveOrganization.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from 'react';
+import { useQuery, useMutation } from '@apollo/client';
+import { ORGANIZATIONS_LIST } from 'GraphQl/Queries/Queries';
+import { REMOVE_MEMBER_MUTATION } from 'GraphQl/Mutations/mutations';
+import { Button, Modal, Form, Spinner, Alert } from 'react-bootstrap';
+import { useParams, useNavigate } from 'react-router-dom';
+import { getItem } from 'utils/useLocalstorage';
+
+const LeaveOrganization = (): JSX.Element => {
+  const navigate = useNavigate();
+  const { orgId: organizationId } = useParams();
+
+  const userEmail = getItem('Talawa-admin', 'email');
+  const userId = getItem('Talawa-admin', 'userId');
+
+  const [email, setEmail] = useState('');
+  const [error, setError] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [showModal, setShowModal] = useState(false);
+  const [verificationStep, setVerificationStep] = useState(false);
+
+  const {
+    data: orgData,
+    loading: orgLoading,
+    error: orgError,
+  } = useQuery(ORGANIZATIONS_LIST, {
+    variables: { id: organizationId },
+  });
+
+  const [removeMember] = useMutation(REMOVE_MEMBER_MUTATION, {
+    onCompleted: () => {
+      alert('You have successfully left the organization!');
+      setShowModal(false);
+      navigate(`/user/organizations`);
+      window.location.reload();
+    },
```

Comment:
After leaving the organization it is navigated to user/organizations and then the org join button still says **visit** instead of **join** that is the reason i put a reload there should remove it or if you suggest something else that would be really helpful

Reply:
i have used the Apollo's refetchQueries option but it didn't work

Reply:
I tried again and refetchQueries works fine :)